frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -1943,7 +1984,8 @@
                     "method_name": vm.method_name,
                     "method_description": vm.method_description,
                     "project_url": vm.project_url,
-                    "publication_url": vm.publication_url
+                    "publication_url": vm.publication_url,
+                    "submission_metadata": JSON.stringify(vm.submissionMetaAttributes[vm.submissionMetaAttributeIndex].attributes)
```

Comment:
@burnerlee Is this correct? `submission_metadata` will include all submission meta attributes right? not just a single one

Reply:
yes, actually submissionMetaAttributes is this type of an object:
```json
[{phaseId: 39, attributes: Array(4)},{phaseId: 40, attributes: Array(4)}]
```
which means that it holds info for both the phases related to that challenge. So we first find out the index in this for the phase of the particular challenge, then we access it's attributes which contains all the submission meta attributes for that challenge surely.

Reply:
@burnerlee are you storing values edited values in `vm.submissionMetaAttributes` ? If I remember correctly, that variable is just being used to render fields on the UI and not to store and submit the data. Can you confirm this by making some edit in one of submissions and switching to `Submit` without refresh to see the default values are not being messed up.

Reply:
@Ram81 sorry i didn't get it correctly what do you want me to confirm. Should I edit the initial values and submit it? and then without refreshing check which values?

Reply:
actually, maybe it was not used before to store the values entered by the user. But I saw that it contained a field value for each attribute, so I stored the previous values of attributes when the dialog box just appears, and later update this values as per values entered by user. And i have used this value for submission as i'd made a change in updatesubmissionmetadata in this PR itself for submitting the new values for attributes

Reply:
@burnerlee using this variable to store values is incorrect. If you make more than 2 submissions and go to my submissions page and edit one of them you'll see all submissions will have same values for meta attributes because you are using single variable to store everything. Can you please fix this. You can check how we are submitting data for the default meta attributes and use similar approach to submit meta attributes. You can use `vm. submissionMetaAttributes` to show fields on the modal but don't store the meta attributes value of submission in that variable.

Reply:
@Ram81  `vm. submissionMetaAttributes` is unique for each submission and appears when we open the edit dialog box for each submission. I understand what you are saying but if I am changing a single submission using this code, it only changes the value for the particular submission, not for all. I've checked this for the code uploaded currently. Could you please try it out once and upload some screenshot of the error you're mentioning

Reply:
I'll fix the code if it has to be done with a new variable. However, it seems fine to me considering the functioning